### ISSUE TITLE
A very interesting job. What is the partitioning method for the dataset? What is the ratio of training set to test set? For example, office_home

### ISSUE THREAD

**Issue by leejq666** [BaiShuanghao/Prompt-based-Distribution-Alignment#2]
What is the ratio of training set to test set? For example, office_home

**Comment by BaiShuanghao** [comment]
We just follow the splits from the dassl library. 
You can see the ratio when the code running.
